src/io/tradetrust/certificate-of-origin/1.0/schema.json

```diff
@@ -4,7 +4,43 @@
   "type": "object",
   "title": "Description of data expected for certificate of origin documents",
   "required": ["iD", "supplyChainConsignment"],
-  "additionalProperties": true,
+  "definitions": {
+    "ImporterExporter": {
```

Comment:
hum if we can't come with a better name, maybe better to inline this. COuld something like Person be used instead ?

Reply:
i have change the naming to CompanyDetails, and saw issuer and manufacturer are using the same property schema but non required thus i split the definitions to "RequiredCompanyDetails" and "CompanyDetails"